staging/src/k8s.io/apiserver/pkg/storage/testing/utils.go

```diff
@@ -197,7 +197,7 @@ func testCheckNoMoreResults(t *testing.T, w watch.Interface) {
 	select {
 	case e := <-w.ResultChan():
 		t.Errorf("Unexpected: %#v event received, expected no events", e)
-	case <-time.After(time.Second):
+	case <-time.After(300 * time.Millisecond):
```

Comment:
@wojtek-t @aojea I assume you have the same concern here as in https://github.com/kubernetes/kubernetes/pull/123897#discussion_r1521558380, right ?

Reply:
Yes - let's not get below 1s.

Reply:
I still think that your concern is not applicable here because the tests are run one after another, which means that there is no other actor that manipulates the database. This, in turn, means that what has been added to the database must be returned in the watch. This, in turn, means that we can wait 300ms for any additional data.